packages/itmat-ui-react/src/components/users/userDetails.tsx

```diff
@@ -158,8 +169,8 @@ export const EditUserForm: React.FunctionComponent<{ user: (IUserWithoutToken &
                             <Input disabled />
                         </Form.Item>
                         <Form.Item name='organisation' label='Organisation'>
-                            <Select disabled>
-                                {Object.entries(sites).map((site) => <Select.Option key={site[0]} value={site[0]}>{site[1]}</Select.Option>)}
+                            <Select>
+                                {Object.entries(orgsMap).map((orgsMap) => <Select.Option key={orgsMap[0]} value={orgsMap[0]}>{orgsMap[1]}</Select.Option>)}
```

Comment:
Maybe it wasn't worth creating `orgsMap` if you're converting it back to an array 😄